test/mocha/10-restrictions.js

```diff
@@ -154,8 +154,41 @@ describe('restrictions', function() {
       secondRestriction.restriction.methodOptions.duration.should.equal('P30D');
     });
 
+  it('should get a restriction by id', async function() {
+    const id = await generateId();
+    await restrictions.insert({
+      restriction: {
+        id,
+        zone: ZONES.TWO,
+        resource: RESOURCES.STRAWBERRY,
+        method: 'limitOverDuration',
+        methodOptions: {
+          limit: 1,
+          duration: 'P30D'
+        }
+      }
+    });
+    const getRestriction = await restrictions.get({id});
+    const expectedRestriction = {
```

Comment:
I suggest defining this as `mockRestriction` (to include the generated `id`) above the insert and simply asserting that the restriction that comes out of the database deep.equals the one that went in.